netlify.toml

```diff
@@ -0,0 +1,9 @@
+[build]
+  command = "npm run build"
```

Comment:
Nit: I usually don't see these lines indented, e.g. [TOML example](https://github.com/toml-lang/toml#example).

Reply:
I see that Netlify does indent them in their sample TOML.

Reply:
I removed the indentation in the end